test/unit/config/matchers/to-match-style-diff-snapshot.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+const snapshotDiff = require( 'snapshot-diff' );
+
+const serialize = ( obj ) => JSON.stringify( obj, undefined, '\t' );
+
+/**
+ * @param {Element} received
+ * @param {Element} expected
+ * @param {string} testName
+ */
+function toMatchStyleDiffSnapshot( received, expected, testName ) {
+	const receivedStyles = serialize(
+		window.getComputedStyle( received )._values
+	);
+	const expectedStyles = serialize(
+		window.getComputedStyle( expected )._values
+	);
+	const difference = snapshotDiff( receivedStyles, expectedStyles );
+	return snapshotDiff.toMatchDiffSnapshot.call(
```

Comment:
Did you check what this function returns? As far as I remember a matcher is an object so maybe you can remove `//` at this stage.

Reply:
See https://github.com/WordPress/gutenberg/blob/master/packages/jest-console/src/matchers.js

It was created before Jest added some internal references to the API that is used there but it probably still works very similar.

Reply:
Ooo I did not check that! I'll look and see if I can do something about it there.